code/modules/mob/living/carbon/human/species_types/furry/tabaxi.dm

```diff
@@ -57,7 +57,7 @@
 		ORGAN_SLOT_LIVER = /obj/item/organ/liver,
 		ORGAN_SLOT_STOMACH = /obj/item/organ/stomach,
 		ORGAN_SLOT_APPENDIX = /obj/item/organ/appendix,
-		ORGAN_SLOT_TAIL = /obj/item/organ/tail/cat,
+		//ORGAN_SLOT_TAIL = /obj/item/organ/tail/cat, //Commenting out due to use of customizer organs.
```

Comment:
Is this actually correct? How do other customizer-using organs handle this?

Reply:
Yup, check the tiefling code, for example.